angular/polyfills-es5.58980aaab917c5b77038.js

```diff
@@ -0,0 +1 @@
astIndex;a(s,0)||(i.lastIndex=0);var f=c(i,u);return a(i.lastIndex,s)||(i.lastIndex=s),null===f?-1:f.index}]}))},g69M:function(t,e,n){var r=n("wA6s"),o=n("rG8t"),i=n("TzEA").f;r({target:"Object",stat:!0,forced:o((function(){return!Object.getOwnPropertyNames(1)}))},{getOwnPropertyNames:i})},g9hI:function(t,e){t.exports=!1},gXAK:function(t,e,n){"use strict";var r=n("wA6s"),o=n("uoca");r({target:"String",proto:!0,forced:n("d8Sw")("big")},{big:function(){return o(this,"big","","")}})},gke3:function(t,e,n){"use strict";var r=n("wA6s"),o=n("kk6e").filter,i=n("lRyB"),a=n("w2hq"),c=i("filter"),u=a("filter");r({target:"Array",proto:!0,forced:!c||!u},{filter:function(t){return o(this,t,arguments.length>1?arguments[1]:void 0)}})},gn9T:function(t,e,n){"use strict";var r={}.propertyIsEnumerable,o=Object.getOwnPropertyDescriptor,i=o&&!r.call({1:2},1);e.f=i?function(t){var e=o(this,t);return!!e&&e.enumerable}:r},"hH+7":function(t,e,n){var r=n("ezU2");t.exports=function(t){if("number"!=typeof t&&"Number"!=r(t))throw TypeError("Incorrect invocation");return+t}},"hN/g":function(t,e,n){"use strict";n.r(e),n("pDpN")},hdsk:function(t,e,n){"use strict";var r,o=n("ocAm"),i=n("8aNu"),a=n("M7Xk"),c=n("wdMf"),u=n("DAme"),s=n("6XUM"),f=n("XH/I").enforce,l=n("yaK9"),p=!o.ActiveXObject&&"ActiveXObject"in o,h=Object.isExtensible,v=function(t){return function(){return t(this,arguments.length?arguments[0]:void 0)}},d=t.exports=c("WeakMap",v,u);if(l&&p){r=u.getConstructor(v,"WeakMap",!0),a.REQUIRED=!0;var g=d.prototype,y=g.delete,m=g.has,b=g.get,k=g.set;i(g,{delete:function(t){if(s(t)&&!h(t)){var e=f(this);return e.frozen||(e.frozen=new r),y.call(this,t)||e.frozen.delete(t)}return y.call(this,t)},has:function(t){if(s(t)&&!h(t)){var e=f(this);return e.frozen||(e.frozen=new r),m.call(this,t)||e.frozen.has(t)}return m.call(this,t)},get:function(t){if(s(t)&&!h(t)){var e=f(this);return e.frozen||(e.frozen=new r),m.call(this,t)?b.call(this,t):e.frozen.get(t)}return b.call(this,t)},set:function(t,e){if(s(t)&&!h(t)){var n=f(this);n.frozen||(n.frozen=new r),m.call(this,t)?k.call(this,t,e):n.frozen.set(t,e)}else k.call(this,t,e);return this}})}},hmpk:function(t,e){t.exports=function(t){if(null==t)throw TypeError("Can't call method on "+t);return t}},i85Z:function(t,e,n){var r=n("U+kB");t.exports=r&&!Symbol.sham&&"symbol"==typeof Symbol.iterator},ipMl:function(t,e,n){var r=n("F26l");t.exports=function(t,e,n,o){try{return o?e(r(n)[0],n[1]):e(n)}catch(a){var i=t.return;throw void 0!==i&&r(i.call(t)),a}}},jnLS:function(t,e,n){var r=n("hmpk"),o="["+n("xFZC")+"]",i=RegExp("^"+o+o+"*"),a=RegExp(o+o+"*$"),c=function(t){return function(e){var n=String(r(e));return 1&t&&(n=n.replace(i,"")),2&t&&(n=n.replace(a,"")),n}};t.exports={start:c(1),end:c(2),trim:c(3)}},kIOX:function(t,e,n){var r=n("ocAm"),o=n("OjQg"),i=n("nP0K"),a=n("aJMj");for(var c in o){var u=r[c],s=u&&u.prototype;if(s&&s.forEach!==i)try{a(s,"forEach",i)}catch(f){s.forEach=i}}},kP9Y:function(t,e,n){var r=n("wA6s"),o=n("4GtL"),i=n("A1Hp")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found non-literal argument to RegExp Constructor](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5238085)